src/owo.js

```diff
@@ -1,34 +1,37 @@
-
 const { createReadStream } = require('fs');
 const Endpoints = require('./Endpoints');
 const request = require('superagent');
 const OwOError = require('./owoError');
 
-/*
- * Creates a new class
- * @prop {key} key your whats-th.is API Key.
- */
+/**
+  * OwO Client
```

Comment:
I still don't get what changes you want from this. I followed the standard, documenting constructor on the constructor, and class description as the top doc comment.